src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For now I think msbuild is a good way to go. Maybe in the future @gep13 wants to convert this to cakebuild? But this is just a note for us to start thinking about whether we want to go that road or not.